index.js

```diff
@@ -201,21 +200,24 @@ function allocAndEncode(value) {
 }
 
 function decode_type(type, buffer, start, len) {
```

Comment:
Replacing this with a `switch` means we can `throw`, we can't `throw` from a ternary inline condition.

Reply:
Switch is must easier to read as well

Reply:
And this should fix https://github.com/ssbc/bipf/issues/9